tests/benchmarks/peripheral_stress_test/src/load_thread.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(load, LOG_LEVEL_INF);
+
+#include "common.h"
+
+static struct k_timer load_timer;
+static bool load_active = true;
```

Comment:
should it be atomic? volatile? to communicate from interrput/callback into main thread?

Reply:
I changed to atomic